modules/twinklespeedy.js

```diff
@@ -1481,6 +1499,10 @@ Twinkle.speedy.callbacks = {
 				text = text.replace(/\{\{(mtc|(copy |move )?to ?commons|move to wikimedia commons|copy to wikimedia commons)[^}]*\}\}/gi, '');
 			}
 
+			if (params.requestsalt) {
+				code = code + '\n{{salt}}';
```

Comment:
Okay well I've gotten crickets regarding my [g10 question](https://github.com/azatoth/twinkle/pull/789#pullrequestreview-341554279), so I think we should try to get the tag to show up on g10 pages.  This should do it I believe:

```suggestion
				if (params.normalizeds.indexOf('g10') === -1) {
					code = code + '\n{{salt}}';
				} else {
					code = '{{salt}}\n' + code;
				}
```